### PR TITLE
Misc. espresso configuration options

### PR DESCRIPTION
**Pull request by nomaxg**
Add hotShotAddress argument, set the l2 chain config with the correct espresso flag

### CODE REVIEW

File: scripts/config.ts [hunk 1]
```diff
@@ -332,6 +332,7 @@ function writeL2ChainConfig(argv: any) {
 		"eip150Block": 0,
 		"eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
 		"eip155Block": 0,
+                "espresso": argv.espresso,
```
Comment (ImJeremyHe) [inline scripts/config.ts:335]: I suddenly thought of a question: if we modify the chain config here, does that mean we can't use integration for the already running chain? This seems inappropriate.

Comment (nomaxg) [reply scripts/config.ts:335]: What do you mean exactly? I think that we plan on running our integration exclusively in espresso mode, we just include the flag to allow for backwards compatibility if need be

Comment (ImJeremyHe) [reply scripts/config.ts:335]: Ah ok. Anyway it seems not a big deal here since we can move this flag to validators' or sequencers' config at any time.

**Review by ImJeremyHe — APPROVED** [review]
LGTM

**Pull request merged**